suites/quincy/rbd/tier-2_rbd_upgrade_5x_to_6x.yaml

```diff
@@ -26,7 +26,7 @@ tests:
                   service: cephadm
                   args:
                     rhcs-version: 5.3
-                    release: "tier-0"
+                    release: "rc"
```

Comment:
Thanks @krishnaramaswamy for the PR
can you please test these changes and update test result?